include/deal.II/fe/mapping_fe_field.h

```diff
@@ -653,6 +656,19 @@ class MappingFEField : public Mapping<dim, spacedim>
   friend class MappingFEField;
 };
 
+/**
+ * @copydoc MappingFEField<dim, spacedim, VectorType>
```

Comment:
Same as in #11189: doxygen doesn't like template parameters in copydoc targets. Do you know how to make this work? Otherwise I'll simply copy the doc.